doc-content/enterprise-only/getting-started-kogito/proc-kogito-remote-process-decision-integrating.adoc

```diff
@@ -0,0 +1,86 @@
+[id="proc-kogito-remote-process-decision-integrating_{context}"]
+= Integrating process and decision microservices in {KOGITO} using the remote method
+
+[role="_abstract"]
+The remote method uses REST endpoints from the decision microservices to integrate the process and decision microservices. In the REST endpoints, the process microservice can use BPMN components to send HTTP requests, such as a service task or a REST work item.
+
+The remote method does not require the process and decision microservices to run in the same application, since the communication is performed using the network.
```

Comment:
Change since to because and search for since in the doc.

Reply:
Done